Tasks/SonarQubePostTest/SonarQubeHelper.ps1

```diff
@@ -0,0 +1,93 @@
+function InvokeMsBuildRunnerPostTest
+{
+	$bootstrapperPath = GetBoostrapperPath
+	$arguments = GetMsBuildRunnerPostTestArgs
+
+	Write-Verbose -Verbose "Executing $bootstrapperPath with arguments $arguments"
+	Invoke-BatchScript $bootstrapperPath -Arguments $arguments
+}
+
+function GetBoostrapperPath
+{
+	$bootstrapperPath = GetTaskContextVariable "bootstrapperPath" 
```

Comment:
Presumably this task context variable is available across steps i.e. visible to all intermediate steps.
Use a namespace-like naming scheme that gives an indication about which step(s) it relates to e.g. sonarqube.bootstrapperPath / msbuild.sonarqube.bootstrapperPath